cmd/insolard/api.go

```diff
@@ -48,3 +49,36 @@ func initAPIInfoResponse() (map[string]interface{}, error) {
 		"migrationDaemonMembers": migrationDaemonMembersStrs,
 	}, nil
 }
+
+// initAPIOptions creates options object, that contains application-specific settings for api component.
```

Comment:
description is too common. Maybe: `initAPIOptions` return default API configuration for platform